pkg/network/go/bininspect/dwarf.go

```diff
@@ -0,0 +1,312 @@
+package bininspect
+
+import (
+	"debug/dwarf"
+	"debug/elf"
+	"errors"
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/network/go/dwarfutils"
+	"github.com/DataDog/datadog-agent/pkg/network/go/dwarfutils/locexpr"
+	"github.com/go-delve/delve/pkg/dwarf/godwarf"
+	"github.com/go-delve/delve/pkg/dwarf/loclist"
+)
+
+// dwarfInspector is used to keep common data for the dwarf inspection functions.
+type dwarfInspector struct {
```

Comment:
not sure I understand - so you mean separating the struct &  it's functions from functions that do not belong to the struct? (like the InspectWithDWARF Func)

Reply:
That's the other way of looking at it :)

Reply:
I think that creating another file here is a bit confusing since the functions (even though they do not belong to the struct) are only being used here

Reply:
That's why I've also suggested the option to keep them together so it will be in more logical order. Anyway it's just an advice so feel free to resolve the comment :)